### PR TITLE
Add version history for 5.5.2

### PR DESCRIPTION
**Pull request by dgault**
See https://github.com/openmicroscopy/bioformats/milestone/53?closed=1
for list of included PRs.

I have not yet included entries for open PR's: https://github.com/openmicroscopy/bioformats/pull/2868 https://github.com/openmicroscopy/bioformats/pull/2867 https://github.com/openmicroscopy/bioformats/pull/2658

### CODE REVIEW

File: docs/sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,45 @@
 Version history
 ===============
 
+5.5.2 (2017 June 15)
+-------------------
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]:
```
     [exec] Warning, treated as error:
     [exec] /home/travis/build/openmicroscopy/bioformats/docs/sphinx/about/whats-new.rst:5: (WARNING/2) Title underline too short.
```

File: docs/sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+
+File format improvements:
+
+* Olympus FluoView FV1000
+   - fix for ``java.lang.ArrayIndexOutOfBoundsException`` caused by filter names
+     of "---"
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: Add acknowledgement of Stefan Helfrich contribution

File: docs/sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,45 @@
...
+   - refactor channel metadata population and make use of ``DataTools`` utility functions
+* Zeiss CZI
+   - fix detection of Z line scans that caused incorrect dimensions in certain filesets
+   - improved exception handling of truncated/invalid files 
+* Veeco AFM
+   - fix for reading tiled images where Y is greater than 0. ``VeecoReader`` flips the 
+     image retrieved using the HDF service vertically to match the expected origin
+* Hamamatsu ndpi
+   - performance improvement to prevent potential memory leak by ensuring all ``TiffParser`` 
+     streams are closed
```
Comment (simleo) [inline docs/sphinx/about/whats-new.rst]: I'd also simplify this one by removing "performance improvement to"

File: docs/sphinx/about/whats-new.rst [hunk 4]
```diff
@@ -1,6 +1,45 @@
...
+
+Bug fixes:
+
+* OMEXMLServiceImpl
+   - improved exception handling in ``OMEXMLServiceImpl`` to deal with potential
```
Comment (simleo) [inline docs/sphinx/about/whats-new.rst]: Remove "in ``OMEXMLServiceImpl``", it's implicit due to the above header

File: docs/sphinx/about/whats-new.rst [hunk 5]
```diff
@@ -1,6 +1,45 @@
 Version history
 ===============
 
+5.5.2 (2017 June 15)
+--------------------
+
+File format improvements:
```
Comment (simleo) [inline docs/sphinx/about/whats-new.rst]: This section also lists bug fixes, perhaps it can be changed to "File format fixes and improvements"

**Comment by simleo** [comment]
Looks good to merge.

**Pull request merged**